docs/source/en/deepspeed.md

```diff
@@ -1141,6 +1141,52 @@ Using multiple GPUs with ZeRO-3 for generation requires synchronizing the GPUs b
 
 For Transformers>=4.28, if `synced_gpus` is automatically set to `True` if multiple GPUs are detected during generation.
 
+### Non-Trainer Sequence Parallelism
+DeepSpeed sequence parallelism, also known as [DeepSpeed Ulysses](https://github.com/microsoft/DeepSpeed/blob/master/blogs/deepspeed-ulysses/README.md), is compatible with HuggingFace Transformers by adding 'sequence_parallel_size' and 'data_parallel_size' to the DeepSpeed configuration. Additionally, it's required that the user’s script correctly shard the input data along the sequence dimension.
```

Comment:
Can we also explain why anyone would want to use this?

Reply:
(I don't mean this as agresssive! More like "if you want to do xxx or yyy this is for you! vs deepspeed normal!)

Reply:
@ArthurZucker, it comes down to user preference. We’ve noticed users utilizing both options from the long list of issues in the DeepSpeed repo. Additionally, there is another PR (number 2877, I think) in progress to support the Accelerate version. At the time this PR was submitted, it was agreed that the pending PR 2877 would leverage this PR to provide support for the Accelerate trainer version. cc: @muellerzr

Reply:
> 我们还可以解释一下为什么有人想使用它吗？

it is very import , shy donot we not need ? for long context train , it is needs